src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -45,7 +45,7 @@ vprintln(args...) = nothing
 using LinearAlgebra
 import Base: *
 import Base.size, Base.adjoint, Base.Array
-
+import DFTK:zeros_like
```

Comment:
Should not be needed.

Reply:
Hum, you are right. But now I'm confused as to why this type of import is required in the tests (such as [here](https://github.com/JuliaMolSim/DFTK.jl/blob/a96f551ed78a47f6cbd5280fedf08f6a7443d31e/test/adaptive_damping.jl#L3)). Is this because the file `lobpcg_hyper_imp` is under `src`, which is also where we define all the functions which be accessed through the DFTK package (in the [DFTK.jl file](https://github.com/JuliaMolSim/DFTK.jl/blob/a96f551ed78a47f6cbd5280fedf08f6a7443d31e/src/DFTK.jl)), whereas the tests are in an other, possibly independent directory?

Reply:
The tests run the code "from the outside" in a new environment to emulate how DFTK would be used. That's why you need to import not exported functions, but inside DFTK itself you don't need to import module-internal stuff.

Reply:
The directory is just for us humans. Julia does not care about this.

Reply:
Alright, I see now. I think I was expecting some sort of internal imports inside DFTK, but they are not needed. Thanks!